src/components/LayoutGrid.vue

```diff
@@ -28,6 +29,13 @@ import { useViewStore } from '../store/views';
 
 export default defineComponent({
   name: 'LayoutGrid',
+  methods: {
+    onFocusView(id: string, type: string) {
```

Comment:
👍️ Good

Also, when I scroll with mouse wheel on a view, then press the arrow keys, its not always the same view that gets its slice changed.  So maybe setActiveView on scroll events too?